src/content/docs/usage/workspaces.mdx

```diff
@@ -32,10 +32,10 @@ daytona create
 ```
 
 <Aside type="tip">
-You can open a new Workspace in an [IDE](/docs/usage/ide) by using the `--code` flag.
+You can skip automatically opening the Workspace in an [IDE](/docs/usage/ide) by using the `--no-ide` flag.
```

Comment:
I'd maybe remove this entire `Aside`

Reply:
Thought about it but auto-open might be annoying for some so I think we should keep it.

Reply:
id keep the no-ide flag here